NEWS

```diff
@@ -1,3 +1,25 @@
+22-Sep-2017 IGNITION v0.19.0
+
+  Features
+
+    - Add support for cloudstack network metadata
+    - Add blackbox tests for tfpt urls
+    - Removed dependency on kpartx for blackbox tests
+
+  Changes
+
+    - Stop adding extra quotes around GECOS field when creating users
+    - Use a private tmpdir subdirectory hen running blackbox tests
+
+  Bug Fixes
+
+    - Fix regression in validation logic causing inaccurate line and column
+      reporting
+    - Fix regression in validation logic where json syntax errors are not
+      reported correctly
+    - Add warnings for links and directories if a non-existent filesystem is
```

Comment:
`for links and directories` seems unclear.  `Add warning if non-existent filesystem is specified when creating links or directories`?